pkg/api/controllers/routes.go

```diff
@@ -18,14 +18,15 @@ func (s *Server) initRoutes() {
 	chandler := h.NewChannelHandler()
 	s.Mux.Handle("/channels", rest.HandleCRUD(chandler))
 
-	cthandler := h.NewTypeHandler()
-	s.Mux.Handle("/types", rest.HandleCRUD(cthandler))
+	thandler := h.NewTypeHandler()
+	s.Mux.Handle("/types", rest.HandleCRUD(thandler))
 
 	aliasHandler := h.NewAliasHandler()
 	s.Mux.Handle("/alias", rest.HandleCRUD(aliasHandler))
 
 	brokersHandler := h.NewBrokerHandler()
-	s.Mux.Handle("/brokers", brokersHandler.HandleGet())
+	s.Mux.Handle("/brokers", brokersHandler.HandleGet().Get().JSON())
+	s.Mux.Handle("/brokers/kafka", brokersHandler.KafkaHandler().Post().JSON())
```

Comment:
Should use the constant defined on `cmd/insprd/memory/brokers/models.go` instead of hardcode "kafka"